src/pages/invite/InviteError.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div :class="getClass">
+    <Card class="content">
+      <Paragraph>You have not been invited to this workspace.</Paragraph>
+      <Button @click="logout">switch account</Button>
+      <Break />
+      <Small>
+        currently logged in as:
+        <Break />
+        {{ user.name }} ({{ user.email }})
+      </Small>
+    </Card>
+  </div>
+</template>
+
+<script>
+import Component from '/components/sharedScripts/component'
+import gql from 'graphql-tag'
+
+export default new Component({
+  name: 'InviteError',
+  apollo: {
+    user: {
+      query: gql`
+        {
+          user {
+            name
+            callname
+            email
+            id
+          }
+        }
+      `,
+      error(err) {
+        if (
+          err.message.replace('GraphQL error:', '').trim() === 'NOT_LOGGED_IN'
+        ) {
+          this.openLogin()
+          return true
+        } else throw Error(err)
+      },
+      fetchPolicy: 'network-only',
+    },
+  },
+  data: {
+    user: {
+      name: '',
+      email: '',
+    },
+  },
+  methods: {
+    logout: function() {
+      this.$apollo.provider.clients.auth
+        .mutate({
+          mutation: gql`
+          mutation {
+            logout(token: "${localStorage.getItem('sessionToken')}")
+          }
+        `,
+        })
+        .then(() => {
+          localStorage.removeItem('sessionToken')
```

Comment:
This is again the same code as in two other places, as I've noted there.

Reply:
done bffe2d1